Xamarin.Forms.ControlGallery.iOS/AppDelegate.cs

```diff
@@ -162,6 +162,8 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
 			MessagingCenter.Subscribe<NativeBindingGalleryPage>(this, NativeBindingGalleryPage.ReadyForNativeBindingsMessage, AddNativeBindings);
 
 			LoadApplication(app);
+			// uncomment to set default button title color for normal state
+			//UIButton.Appearance.SetTitleColor(UIColor.Red, UIControlState.Normal);
```

Comment:
Custom renderer would probably work. Another option might be to use MessagingCenter to communicate from the test to AppDelegate and update the proxy appearance, then reset it after the test has run. I'm not sure whether that would work, but it might be easier than creating a custom renderer.